frontend/packages/kubevirt-plugin/src/constants/vm/workload-profile.ts

```diff
@@ -27,6 +27,14 @@ export class WorkloadProfile extends SelectDropdownObjectEnum<string> {
     order: 3,
   });
 
+  static readonly SAP_HANA = new WorkloadProfile('saphana', {
+    // t('kubevirt-plugin~High-performance')
```

Comment:
Shouldn't these strings be updated to match the string below them?

Reply:
btw/ why do you need it ?
SAP hanna is not a regular workoad -
For example you can't choose tiny / large / sap-hanne only tiny/large
I think adding it here may couse problems in the wizard, ? for example it will start pop up as a workoad option ?

you should be carful not to mix it with regular workoads (e.g. tiny / small / large ... )
it's only a way to know it's a sap hanna template it's not a real workload

Reply:
> Shouldn't these strings be updated to match the string below them?

You're right, will be fixed

>SAP hanna is not a regular workoad -
>For example you can't choose tiny / large / sap-hanne only tiny/large
>I think adding it here may couse problems in the wizard, ? for example it will start pop up as a workoad option ?

since we are using `workload.template.kubevirt.io/saphana='true'` once it's inserted to the template the UI breaks when trying to get this workload profile, so that's why it's needed, or maybe another we can go with another approach if you have any idea :)

Reply:
> once it's inserted to the template the UI breaks when trying to get this workload profile

ok, fixing the UI to handle any workload is out of scope for this PR, lets leave it for now, in the bigger epic we will need to fix the UI to not break on un expected workload strings.